packages/api/cms-api/src/access-log/access-log.module.ts

```diff
@@ -0,0 +1,35 @@
+import { DynamicModule, Global, Module } from "@nestjs/common";
+import { APP_INTERCEPTOR } from "@nestjs/core";
+import { CurrentUser } from "src/user-permissions/dto/current-user";
+
+import { FILTER_AUTHENTICATED_REQUEST } from "./access-log.constants";
+import { AccessLogInterceptor } from "./access-log.interceptor";
+
+export type FilterAuthenticatedRequest = ({ user, authHeader }: { user?: CurrentUser; authHeader?: string }) => boolean;
+
+interface AccessLogModuleOptions {
+    filterAuthenticatedRequest?: FilterAuthenticatedRequest;
```

Comment:
I think we should consider adding an even more generic `filterRequests: (req: Req) => boolean` that allows us to filter any sort of request. Mostly this feature will be used to exclude request with the API_PASSWORD, but there may be even more specific requirements in some projects.

Reply:
My suggestion: `filterRequests: (currentUser: CurrentUser, req: Req) => boolean`